ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
+import { OptimoBlock } from '#models/types/optimo';
+import headings from '#app/legacy/containers/Headings';
+import Blocks from '#app/legacy/containers/Blocks';
+import paragraph from '#app/legacy/containers/Paragraph';
+import Text from '#app/components/Text';
```

Comment:
Can we use relative path imports instead of using aliases (per https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/React-Components.stories.mdx#typescript-for-frontend-components)

Reply:
This is a good point. I noticed we were already using aliases in the NextJs app, but didn't know why. I posted a question in the Si tech channel. It seems like there is an arguement to use alias in the the NextJS app since:

_There was a technical problem in the Express app that meant trying to navigate to an aliased path in VSCode would often not work. This isn't the case when using TypeScript aliasing in the tsconfig._

What are your thoughts? If we do go this way then we should update the component standards to reflect it.